server/server.go

```diff
@@ -36,21 +36,24 @@ const (
 	// dynamically generated Juju GUI configuration file.
 	jujuVersion       = "2.2.0"
 	legacyJujuVersion = "1.25.7"
+
+	// webSocketBufferSize holds the frame size for WebSocket messages.
+	webSocketBufferSize = 65536
```

Comment:
Does this have to be kept in sync with Juju? Might be worth a comment to that effect.